pkg/disruptors/pod_test.go

```diff
@@ -358,3 +428,78 @@ func Test_PodGrpcPFaultInjection(t *testing.T) {
 		})
 	}
 }
+
+func Test_VerifyPort(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		title       string
+		namespace   string
+		labels      map[string]string
+		targets     []string
+		targetPort  uint
+		expectError bool
+	}{
+		{
+			title:     "Successful port verification",
```

Comment:
If we remove the `PortValidator` I think these test cases should be moved to the pod helper tests.